Modules/expat/amigaconfig.h

```diff
@@ -7,6 +7,9 @@
 /* Define to 1 if you have the `bcopy' function. */
 #define HAVE_BCOPY 1
 
+/* Define to 1 if you have the <check.h> header file. */
+#undef HAVE_CHECK_H
```

Comment:
we don't even need to carry this file around given the `#elif defined(__amigaos__)` it is included under.